poly-commitment/src/commitment.rs

```diff
@@ -587,6 +590,10 @@ impl<G: CommitmentCurve> SRS<G> {
             });
         }
 
+        for _ in unshifted.len()..num_chunks {
+            unshifted.push(G::zero());
```

Comment:
Is this happening just when `is_zero`? Otherwise I was expecting the `else` clause to push `num_chunks` of them, doesn't it?

Reply:
Seems to work in testing